fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1697,6 +1697,11 @@ columnAliases
 partitionNames
     : TEMPORARY? (PARTITION | PARTITIONS) '(' identifier (',' identifier)* ')'
     | TEMPORARY? (PARTITION | PARTITIONS) identifier
+    | lakePartition
```

Comment:
maybe you should create another partition spec after partition name, because partition name in internal table is not the same as external table 

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/8065637/231904135-928e4d3d-6552-49e8-a763-b798b0cb8730.png">